### PR TITLE
improve some messages in the node subcmds

### PR DESCRIPTION
**Pull request by vieux**
from:

```
root@docker112-node01:~# docker node accept 18f4rxq0hga08
18f4rxq0hga08
18f4rxq0hga08 attempting to accept a node in the swarm.
```

to

```
root@docker112-node01:~# docker node accept 18f4rxq0hga08
attempting to accept node 18f4rxq0hga08 in the swarm.
```

ping @cpuguy83 @dnephin @icecrime

### CODE REVIEW

**Comment by thaJeztah** [comment]
LGTM

**Comment by thaJeztah** [comment]
LGTM

**Comment by icecrime** [comment]
Can we capitalize the message? :innocent:

File: api/client/node/update.go [hunk 1]
```diff
@@ -47,7 +51,6 @@ func runUpdate(dockerCli *client.DockerCli, nodeID string, mergeNode func(node *
 		return err
 	}
 
-	fmt.Fprintf(dockerCli.Out(), "%s\n", nodeID)
```
Comment (dnephin) [inline api/client/node/update.go]: Why this change? `args[0]` is already passed as nodeID, so this message would be the same, and I think it makes more sense to have this in `runUpdate()`

Comment (vieux) [inline api/client/node/update.go]: @dnephin because run update is also used in `node promote/demote/accept` see my example in the PR description, it prints 2 lines

**Comment by aaronlehmann** [comment]
> ```
root@docker112-node01:~# docker node accept 18f4rxq0hga08
attempting to accept node 18f4rxq0hga08 in the swarm.
```

This makes it look like the attempt never succeeded. Can we change it to a message that indicates success after the nodespec update is accepted?

**Comment by vieux** [comment]
@icecrime done

**Comment by vieux** [comment]
@aaronlehmann right now the call is asynchronous, are you suggesting we change it to make it synchronous ?

File: api/client/node/update.go (hunk 1 above)
Comment (dnephin) [inline api/client/node/update.go]: I see. The pattern with the cli functions was to have most of the logic that would have been in the `RunE()` extracted into the `runX()` functions, but those `runX()` weren't really designed to be re-used. I guess it's not a big deal. We would need to refactor this further to support that idea.

**Comment by vieux** [comment]
@aaronlehmann @tonistiigi how about `Node 18f4rxq0hga08 accepted in the swarm.` ?

**Comment by tonistiigi** [comment]
> how about Node 18f4rxq0hga08 accepted in the swarm. ?

@vieux LGTM

**Comment by aaronlehmann** [comment]
LGTM

**Comment by icecrime** [comment]
LGTM
